interp/notation_ops.ml

```diff
@@ -712,6 +712,20 @@ let abstract_return_type_context_notation_constr tml rtn =
   abstract_return_type_context snd
     (fun na c -> NLambda(na,NHole (Evar_kinds.InternalHole, IntroAnonymous, None),c)) tml rtn
 
+let rec push_pattern_binders vars pat =
```

Comment:
New function: please document it.

Reply:
It is a local function. Isn't the name not enough self-explanatory?